From ac82e6802c80841607a48f2907c20380844ecb64 Mon Sep 17 00:00:00 2001
From: John French <jdaviesf7@gmail.com>
Date: Wed, 13 May 2015 00:55:00 -0400
Subject: [PATCH 095/214] Exclude CNAME
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

There’s no reason you’d want to be able to access the CNAME file on the
hosted site.
---
 _config.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/_config.yml b/_config.yml
index e77fe90..cebf496 100644
--- a/_config.yml
+++ b/_config.yml
@@ -74,3 +74,4 @@ exclude:
     - Gemfile.lock
     - LICENSE
     - README.md
+    - CNAME
-- 
2.17.1

